demo.js

```diff
@@ -148,7 +148,11 @@ var showEditJSONModal = function () {
 	$("#jsonContainer").fadeIn();
 
 	// create the editor	
-	var options = {};
+	var options = {
+		mode: 'code',
```

Comment:
Default mode would be code edit mode for JSON. User can also choose to select tree mode if needed from the top navbar of editor.